src/components/NavBar/NavBar.js

```diff
@@ -1,24 +1,30 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, withRouter } from 'react-router-dom';
+import Svg from 'react-inlinesvg';
 import './NavBar.scss';
-import navOption1 from '../../assets/icons/nav-option-1.svg';
-import navOption2 from '../../assets/icons/nav-option-2.svg';
-import navOption3 from '../../assets/icons/nav-option-3.svg';
+import mapIcon from '../../assets/icons/nav/map.svg';
+import placesListIcon from '../../assets/icons/nav/list.svg';
+import favouritePlacesIcon from '../../assets/icons/nav/favourite.svg';
 
-const NavBar = () => {
+const links = [
+  { to: '/', src: mapIcon },
+  { to: '/restaurant-info', src: placesListIcon },
+  { to: '/favourite-places', src: favouritePlacesIcon }
+];
+
+const NavBar = props => {
   return (
     <div className="navbar">
-      <Link to="/">
-        <img src={navOption1} alt="icon-1" className="navbar__icon" />
-      </Link>
-      <Link to="/restaurant-info">
-        <img src={navOption2} alt="icon-2" className="navbar__icon" />
-      </Link>
-      <Link to="/">
-        <img src={navOption3} alt="icon-3" className="navbar__icon" />
-      </Link>
+      {links.map((link, index) => (
+        <Link to={link.to} key={index}>
+          <Svg
+            src={link.src}
+            className={`navbar__icon ${link.to === props.location.pathname ? 'navbar__active' : ''}`}
```

Comment:
what about
```
const {pathname} = useLocation()
...
const classNames = ['navbar__icon', link.to === pathname && 'navbar__active'].filter(Boolean).join(" ")
```

Reply:
it would be good idea to highlight one of the icons also when we're on restaurant details page 🤔 it'd be better to change pathname.includes('...')